Exercise_D and E/assets/style/index.css

```diff
@@ -2,3 +2,54 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+.imgcontainer {
+  display: flex;
+  flex-direction: column;
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
+    url(https://i.imgur.com/ibg3zaN.jpg);
```

Comment:
Use the relative path from this index.css to the image, stored within the image folder.
Great use of linear-gradient to apply the dark overlay!